### PR TITLE
Feature/v3.10.0

### LINKED ISSUE #21: Chirp dependency outdated
Here in the demo, the dependency used in app level build gradle is 3.9.7, which is outdated and is not working anymore

### PR DESCRIPTION
**Pull request by dinuchiriac**
Upgrade demo apps to use the latest ChirpSDK

closes #21 #23

### CODE REVIEW

**Comment by dinuchiriac** [comment]
> 1. Demo app UI title `Chirp Connect` -> `ChirpSDK Demo` or something like.

done @joextodd

File: Java/Demo/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 <resources>
-    <string name="app_name">ConnectDemoApp</string>
+    <string name="app_name">ChirpSDKoApp</string>
```
Comment (joextodd) [inline Java/Demo/app/src/main/res/values/strings.xml]: ChirpSDKoApp?

File: Kotlin/Demo/app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -45,7 +45,7 @@
                 android:layout_height="wrap_content"
                 android:layout_weight="1"
                 android:gravity="center"
-                android:text="Chirp Connect"
+                android:text="Chirp SDK"
```
Comment (joextodd) [inline Kotlin/Demo/app/src/main/res/layout/activity_main.xml]: This one's different to the Java one..

File: Kotlin/Messenger/app/src/main/java/io/chirp/messenger/MainActivity.kt [hunk 3]
```diff
@@ -33,7 +33,7 @@ private const val TAG = "ChirpMessenger"
 
 class MainActivity : AppCompatActivity() {
 
-    private lateinit var chirpConnect: ChirpConnect
+    private lateinit var chirpConnect: ChirpSDK
```
Comment (joextodd) [inline Kotlin/Messenger/app/src/main/java/io/chirp/messenger/MainActivity.kt]: `chirpConnect` still appearing in variable name, just `chirp` is what we have used on the documentation.

**Comment by dinuchiriac** [comment]
> Just one lingering `connect` reference in Messenger.

Done

**Comment by joextodd** [comment]
@dinuchiriac nice, will merge once SDK is released

**Pull request merged**